app/themes/base.js

```diff
@@ -24,32 +24,47 @@ import {
 // and https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-const/src/variables.js
 
 export default {
-  breakpoints: [
-    '13.6em',
-    '30.4em',
-    '47.2em',
-    '64em',
-    '97.6em'
-  ],
+  breakpoints: {
+    mobile: '30em', // 480px,
+    mobileWide: '37.5em', // 600px,
+    tablet: '48em', // 768px,
+    tabletWide: '56.25em', // 900px,
+    desktop: '64em', // 1024px,
+    desktopWide: '90em', // 1440px,
+    desktopXL: '120em' // 1920px
+  },
   space: [
-    '0.25rem',
-    '0.5rem',
-    '1rem',
-    '2rem',
-    '4rem',
-    '8rem',
-    '16rem',
+    '0rem', // [0] SR: to override default allofthespace with noneofthespace
+    '0.25rem', // [1]
+    '0.5rem', // [2]
+    '1rem', // [3]
+    '2rem', // [4]
+    '4rem', // [5]
+    '8rem', // [6]
+    '12rem', // [7]
+    '16rem', // [8]
+    '20rem', // [9]
+    '24rem' // [10]
   ],
   fontSizes: [
-    '0.875rem',
-    '1rem',
-    '1.25rem',
-    '1.5rem',
-    '2.25rem',
-    '3rem',
-    '5rem',
-    '6rem'
+    '0.8rem', // [0] 10pt
```

Comment:
i just did a google image search for those images, but will see if i can find a good article to back it up